testing/pypy/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor:Mihnea Saracin <mihnea.saracin@rinftech.com>
+# Maintainer:Mihnea Saracin <mihnea.saracin@rinftech.com>
+pkgname=pypy
+pkgver=5.7.1
+pkgrel=0
+pkgdesc="pypy2"
+url="https://bitbucket.org/pypy"
+arch="all"
+license="MIT"
+depends="libssl1.0 libcrypto1.0 libffi libbz2 "
```

Comment:
This *should* be automatically inferred by `abuild` as noted at the end of the build log, although I can't double-check since the Travis build is timing out.